### PR TITLE
feat-monterail/vue3-docs-pl#24

### LINKED ISSUE #24: [Docs > Guide > Quick Start] page translation
URL preview:
https://vuejs.org/guide/quick-start.html

File to be translated:
https://github.com/monterail/vue3-docs-pl/blob/main/src/guide/quick-start.md

### PR DESCRIPTION
**Pull request by Baroshem**

### CODE REVIEW

File: src/guide/quick-start.md [hunk 1]
```diff
@@ -354,25 +354,25 @@ We can teach the browser where to locate the `vue` import by using [Import Maps]
...
+[Demo CodePen >](https://codepen.io/vuejs-examples/pen/YzRyRYM)
 
 </div>
 
-You can also add entries for other dependencies to the import map - but make sure they point to the ES modules version of the library you intend to use.
+Możesz również dodać wpisy dla innych zależności do mapy importu - ale upewnij się, że wskazują one na wersję modułów ES biblioteki, której zamierzasz użyć.
 
-:::tip Import Maps Browser Support
-Import Maps is a relatively new browser feature. Make sure to use a browser within its [support range](https://caniuse.com/import-maps). In particular, it is only supported in Safari 16.4+.
+:::tip Obsługa przeglądarki Import Maps
```
Comment (szymonlicau) [inline src/guide/quick-start.md:363]:
```suggestion
:::tip Wsparcie przeglądarek dla Import Maps
```

File: src/guide/quick-start.md [hunk 2]
```diff
@@ -6,27 +6,27 @@ footer: false
...
-- If you prefer a plain HTML setup without any build steps, you can use this [JSFiddle](https://jsfiddle.net/yyx990803/2ke1ab0z/) as your starting point.
+- Jeśli wolisz prostą konfigurację HTML bez żadnych kroków kompilacji, możesz użyć tego [JSFiddle](https://jsfiddle.net/yyx990803/2ke1ab0z/) jako twój punkt startowy.
 
-- If you are already familiar with Node.js and the concept of build tools, you can also try a complete build setup right within your browser on [StackBlitz](https://vite.new/vue).
+- Jeśli znasz już Node.js i koncepcję narzędzi do kompilacji, możesz również wypróbować pełną konfigurację kompilacji bezpośrednio w przeglądarce na [StackBlitz](https://vite.new/vue).
 
-## Creating a Vue Application {#creating-a-vue-application}
+## Tworzenie aplikacji Vue {#creating-a-vue-application}
 
 :::tip Prerequisites
```
Comment (szymonlicau) [inline src/guide/quick-start.md:21]: We should translate this one too I guess 😄

**Review by szymonlicau — CHANGES_REQUESTED** [review]

File: src/guide/quick-start.md (hunk 2 above)
Comment (Baroshem) [inline src/guide/quick-start.md:21]:
```suggestion
:::tip Wymagania wstępne
```

Comment (Baroshem) [reply src/guide/quick-start.md]: Added. Thanks! :)

**Review by szymonlicau — APPROVED** [review]

**Pull request merged**